packages/database/src/migrations/20200925011104-CreateRootMapOverlay-modifies-data.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+import { insertObject, generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const WORLD_MAP_OVERLAY_GROUP = {
+  id: generateId(),
+  name: 'World',
+  code: 'World_Group',
+};
```

Comment:
Create a world overlay group (context: https://github.com/beyondessential/tupaia-backlog/issues/1109#issuecomment-698657929). 

Thanks @kael89 for your input! I totally agree with your suggestion now. I initially thought it was a bit hacky when we discussed but when the implementation comes together I think it's great! I didn't find that I have to do anything dodgy to make this work

Reply:
Good to know! I also had a bit suspicion that this may be "hacky", but the more about it the more it made sense. Eventually, I believe that our "hack" alert was triggered by the fact we will create a group that will not be displayed - but perhaps this is a view, not data model, issue. We could indeed display the root overlay group in the UI, if we chose to!

Reply:
That's nice! I'm not sure about the name though... "World" brings entities to my mind. Perhaps we should name it something else, like "Root"? What do you think?

To keep things simple, the code can be the same as the name (eg "Root", not "Root_Group")

Reply:
Ah yep, I was also picky between World and Root, but not sure why I went with World. Happy to change it to Root!